prow/cmd/autobump/bump.sh

```diff
@@ -66,34 +66,52 @@ main() {
   else
     usage
   fi
-  echo -e "Bumping: 'gcr.io/k8s-prow/' images to $(color-version ${new_version}) ..." >&2
+  echo -e "Bumping: 'gcr.io/k8s-prow/' images to $(color-version "${new_version}") ..." >&2
 
   bumpfiles=($(add_suffix "$(split_on_commas "$COMPONENT_FILE_DIR")"))
   bumpfiles+=("${CONFIG_PATH}")
   if [[ -n "${JOB_CONFIG_PATH}" ]]; then
     bumpfiles+=($(grep -rl -e "gcr.io/k8s-prow/" "${JOB_CONFIG_PATH}"))
   fi
 
-  echo -e "Attempting to bump the following files: ${bumpfiles[@]}" >&2
+  echo -e "Attempting to bump the following files:" >&2
+  IFS=$'\n'; echo -e "${bumpfiles[*]}" >&2
```

Comment:
wouldn't `IFS=$'\n' echo ...` be more appropriate than permanently changing IFS (via the `;`')? also

```sh
for bf in "${bumpfiles[@]}"; do
  echo -e "$bf"
done
```

is IMO easier to understand than working with IFS

Reply:
Changed to for loop. Thanks!